src/main/scala/zio/s3/settings.scala

```diff
@@ -33,6 +33,68 @@ object S3Region {
       case r                                 => Left(InvalidSettings(s"Invalid aws region provided : ${r.id}"))
     }
 
+  val AP_SOUTH_1 = "ap-south-1"
```

Comment:
Can you please explain the use case ? 

Since this already exist in the amazon SDK
```scala
software.amazon.awssdk.regions.Region.AF_SOUTH_1
```

Reply:
seems a bit of a leak to me. also the api allows for a string but that is of type `Region`. Overall not a big deal, but its annoying to have to go fetch it.